README.md

```diff
@@ -27,12 +27,10 @@ Usage: <type>(producer|consumer)
 * **type**: Defines if the the job is producing/consuming to/from Kafka.
 * **sink**: Used only by a Consumer job, it defines the path to the Dataset or folder to which the Spark job appends its streaming output. The latter contain the consumed Avro records from Kafka. The name of the folder is suffixed with the YARN applicationId to deferantiate between multiple jobs writing to the same Dataset. In this example, the sink file contains data from the latest microbatch. The default microbatch period is set to two(2) seconds.
 
-**MainClass** is io.hops.examples.spark.kafka.StreamingExample
+**MainClass** is io.hops.examples.spark.kafka.StructuredStreamingKafka
 
 **Topics** are provided via the Hopsworks Job UI. User checks the *Kafka* box and selects the topics from the drop-down menu. When consuming from multiple topics using a single Spark directStream, all topics must use the same Avro schema. Create a new directStream for topic(s) that use different Avro schemas.
 
-**Consumer groups** are an advanced option for consumer jobs. A default one is set by Hopsworks and a user can add further ones via the Jobservice UI.
-
 Data consumed is be default persisted to the `Resources` dataset of the Project where the job is running.
 
 ### Avro Records
```

Comment:
Change `StructuredStreamingKafka.java` to `StructuredStreamingKafka.scala`

Reply:
done